### PR TITLE
Upgrade auctions bot

### PR DESCRIPTION
**Pull request by drklee3**
Upgrades auction bot to v0.44 and uses grpc for queries and txs.

### CODE REVIEW

File: auction-bot/grpc_client.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	Tm             tmservice.ServiceClient
+	Auction        auctiontypes.QueryClient
+	Cdp            cdptypes.QueryClient
+	Hard           hardtypes.QueryClient
+	Pricefeed      pricefeedtypes.QueryClient
+}
+
+func NewGrpcClient(target string, enableTls bool, cdc codec.Codec) GrpcClient {
+	var secureOpt grpc.DialOption
+	if enableTls {
```
Comment (nddeluca) [inline auction-bot/grpc_client.go:34]: We could include the scheme in the `target`, then choose secure/insecure based on that - example at https://github.com/Kava-Labs/go-tools/blob/master/hard-keeper-bot/main.go#L40.

Does require passing the entire `https://grpc.testnet.kava.io:443` string, but doesn't require a separate parameter

File: auction-bot/data.go [hunk 2]
```diff
@@ -17,46 +23,43 @@ type AssetInfo struct {
...
-	height := info.LatestHeight
-
-	prices, err := client.GetPrices(height)
+	pricesRes, err := client.Pricefeed.Prices(ctxAtHeight(latestHeight), &types.QueryPricesRequest{})
 	if err != nil {
 		return nil, err
 	}
 
-	auctions, err := client.GetAuctions(height)
+	auctionsRes, err := client.Auction.Auctions(ctxAtHeight(latestHeight), &auctiontypes.QueryAuctionsRequest{})
```
Comment (nddeluca) [inline auction-bot/data.go:43]: I believe we still need to paginate through all auctions here

**Review by nddeluca — CHANGES_REQUESTED** [review]
lgtm, I believe the only change that needs to be made is paginating through all auctions

**Review by karzak — APPROVED** [review]
LGTM - tested against incentivized testnet and was working properly

**Review by nddeluca — APPROVED** [review]

**Pull request merged**